spec/features/reviews/create_new_review_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature 'Create a new review' do
+  let(:venue) { FactoryGirl.create(:venue) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  before do
+    visit new_venue_review_path(venue.id)
+  end
+
+  scenario 'User fills out review form correctly' do
```

Comment:
What is the expectation here? The test assertion sentence is missing the assertion. Maybe try something like `scenario 'User fills out review form correctly do` next line `it 'will show the saved rating and success message' do`. There are many ways to organize it, don't feel attached to my suggestion